server/server_comm/test_runner/views.py

```diff
@@ -37,12 +37,13 @@ def post(self, request, flow_id=None):
                 "error": None,
             }
             try:
-                res = self.check_device_connections(flow_id)
-                if res['status'] != 'success':
-                    flow_result["status"] = "failed"
-                    flow_result["error"] = res["message"]
-                    results.append(flow_result)
-                    break
+                # TODO comment back in when this endpoint is implemented
```

Comment:
isn't this endpoint done? :o

Reply:
Was not done when the branch was created, will remove comment:)